### PR TITLE
add action ui for deleting an element - #1125

### PR DESCRIPTION
**Pull request by devbrock**

### CODE REVIEW

**Comment by Fausto95** [comment]
Thanks for sending this PR.
I'd put it right after the `paste` action.

**Comment by devbrock** [comment]
fixes some of #1125 

I tried to add duplication as well, but couldn't figure out how to get that one working.

File: src/components/Actions.tsx [hunk 1]
```diff
@@ -59,6 +59,12 @@ export function SelectedShapeActions({
           {renderAction("bringForward")}
         </div>
       </fieldset>
+      <fieldset>
```
Comment (Fausto95) [inline src/components/Actions.tsx]: I don't think that this is the right way to do it

**Comment by lipis** [comment]
#1142

**Comment by j-f1** [comment]
Isn’t this already in the toolbar on mobile?

File: src/components/Actions.tsx [hunk 2]
```diff
@@ -59,6 +59,12 @@ export function SelectedShapeActions({
...
+        <legend>Layers Actions</legend>
```
Comment (vjeux) [inline src/components/Actions.tsx]: This needs to use translations

Comment (j-f1) [reply src/components/Actions.tsx]: And it should be “Actions” I think.

**Pull request merged**